common/lib/xmodule/xmodule/templates/problem/checkboxes_response.yaml

```diff
@@ -3,21 +3,46 @@ metadata:
     display_name: Checkboxes
     markdown: |
       A checkboxes problem presents checkbox buttons for student input. Students can select more than one option presented.
-      >>Select the answer that matches<<
+      
+      Ideally, compound feedback would be provided for all possible combinations of answers.  In practice, that is extremely difficult for problems with more than 3 options.  A good compromise is to write individual feedback for each option, and compound feedback for common combinations of answers or combinations of answers that address common student misconceptions.  
```

Comment:
Is it possible to linesplit in these yaml files so that it's not so hard to read? (not sure if possible)